docs/changelog-section/overview-changelog.md

```diff
@@ -2,6 +2,7 @@
 id: overview-changelog
 title: About the changelog
 sidebar_label: Overview
+custom_edit_url: null
```

Comment:
Instead of adding this line to each article can we try to remove `editUrl` line 20 in `docusaurus.config.js`? 

Based on my testing this removes the edit link for all pages.

```
"@docusaurus/preset-classic",
{
  docs: {
    // Docs folder path relative to website dir.
    path: "./docs",
    // Sidebars file relative to website dir.
    sidebarPath: require.resolve("./sidebars.json"),
    editUrl: "https://github.com/Open-Attestation/documentation-website/tree/master/website", // <--- this line
  },
```

Reply:
Noted. Updated each `.md` and `.mdx` file to remove `custom_edit_url: null` and updated `docusaurus.config.js`

Reply:
can you remove the empty newlines in the affected files so we don't have so many unnecessary changes within the PR? thanks